Functions/Verbs/msvc90/script.js

```diff
@@ -0,0 +1,25 @@
+include(["Functions", "Engines", "Wine"]);
+include(["Functions", "Net", "Resource"]);
+include(["Functions", "Filesystem", "Extract"]);
+include(["Functions", "Verbs", "luna"]);
+
+Wine.prototype.msvc90 = function() {
```

Comment:
And does it stop working if you replace the dlls of a working msvc90 installation by vcrun2008 ones?